apps/full-site-editing/full-site-editing-plugin/full-site-editing/plugins/editor-template-classes/index.js

```diff
@@ -39,7 +40,7 @@ const EditorTemplateClasses = withSelect( select => {
 		}
 		clearInterval( blockListInception );
 
-		blockList.className = classNames( 'a8c-template-editor', ...templateClasses );
+		blockList.className = classNames( 'a8c-template-editor fse-template', ...templateClasses );
```

Comment:
Maybe `fse-template-part` would be more appropriate now. But this is optional since there are still a lot of `template` usages that could be refactored to `template part`.